doc/api/http2.md

```diff
@@ -3085,6 +3091,7 @@ following additional properties:
 [Compatibility API]: #http2_compatibility_api
 [HTTP/1]: http.html
 [HTTP/2]: https://tools.ietf.org/html/rfc7540
+[HTTP2 Unencrypted]: https://http2.github.io/faq/#does-http2-require-encryption
```

Comment:
A nit: we usually sort references in ASCII order, so this should go after `[HTTP2 Settings Object]` ref.

Reply:
I fixed this by renaming to `[HTTP/2 Unencrypted]` since its not linking to a section anchor but its more like the external link above.